.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "bracketSpacing": false,
+  "singleQuote": true,
```

Comment:
It looks like you intended to use single quotes for the project but all the quotes have been swapped out for doubles. 
If you would like prettier to be used for formatting it should be installed as a dev dependency and setup via a formatting script, such as `npm run-script format` or IDE integration. 
You could then use EsLint to enforce this.